regression/cbmc-incr/timeout.pl

```diff
@@ -0,0 +1,2 @@
+alarm shift @ARGV;
```

Comment:
⛏️ Maybe this script can be moved to a more central place to avoid duplicating it.